### DISCUSSION TITLE
How to set maximum number of simultaneous downloads?

### DISCUSSION THREAD

**Discussion by Nabukodonosor** [Novik/ruTorrent#2286] [Q&A]
The title says it all. I've tried this in my rtorrent.rc file:

`throttle.max_downloads.global.set = 1`

As you can see, it still downloads all of them at the same time:
http://prntscr.com/26ncy33

I would like to set it to maximum 1 download at a time, and the rest of the to wait in queue. Last night I started 6 downoads an wanted to watch the first movie, but only later I found out that all 6 were downloading at the same time, so I had to wait for all 6 do be downloaded.

How can I accomplish this?

**Comment by Nabukodonosor** [comment]
Anyone?

**Comment by n1ce-0ne** [comment]
afaik its not possible

**Comment by Nabukodonosor** [comment]
Aah, that's too bad. Such a simple, but yet useful feature. Every other torrent client has it.

**Reply by n1ce-0ne** [reply]
i know & agree, same issue with duplicate ip's, qbit handles them fine but not rtorrent

**Reply by stickz** [reply]
You can use ip-tables to fix the duplicate ip issue. It's possible to limit the number of connections per IP.

**Reply by n1ce-0ne** [reply]
atm rtorrent can only connect to one instance per ip, i want a way to allow multiple connections to a single ip
